UI/ui_quest.csv

```diff
@@ -103,7 +103,7 @@ Q_Event001_01_MES#0,"""The ARKS are preparing a large-scale operation in respons
 Q_Event001_02_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Naberian tundra."""
 Q_Event001_03_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Naberian ruins."""
 Q_Event001_04_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Amduscian caverns."""
-Q_Event001_05_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Amduscian floating continent."""
+Q_Event001_05_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Amduscian Skyscape."""
```

Comment:
Reading over Synth's last commit, yeah, it should be capitalized. Just seemed odd because the rest of the areas aren't :P